pwa/src/templates/templateParts/mappingForm/MappingFormTemplate.tsx

```diff
@@ -158,11 +260,18 @@ export const MappingFormTemplate: React.FC<MappingFormTemplateProps> = ({ mappin
                   </div>
 
                   {isOpenMapping && (
-                    <div className={styles.triggerButton} onClick={(e) => handleScrollToCreate(e, mappingRefBottom)}>
-                      <Link icon={<FontAwesomeIcon icon={faArrowDown} />} iconAlign="start">
-                        {t("Scroll to create")}
-                      </Link>
-                    </div>
+                    <>
+                      <div onClick={(e) => handleSwitchEditor(e, "mapping")}>
+                        <Link icon={<FontAwesomeIcon icon={faEdit} />} iconAlign="start">
+                          {t(!editMappingInEditor ? "Edit as JSON" : "Edit as Key-Value")}
```

Comment:
I think this button is a bit out-of-place. I've tried to replace it with Tabs, but that didn't really work out either. Maybe using the ActionButton (needs some refactoring, to also allow these actions and icons), resulting in something like (before refactoring):

![image](https://github.com/ConductionNL/gateway-ui/assets/66552932/9c28f429-e136-4d74-82d5-49850fb2f7e3)